mods/win7-alttab-loader.wh.cpp

```diff
@@ -0,0 +1,668 @@
+// ==WindhawkMod==
+// @id              win7-alttab-loader
+// @name            Windows 7 Alt+Tab Loader
+// @description     Loads Windows 7 Alt+Tab on Windows 10.
+// @version         1.0
+// @author          ephemeralViolette, aubymori

```

Comment:
Please pick and use one author and also add `@github`. You can mention other authors in the readme.

Reply:
`@github` is still missing. Also note that the author details must match the submitter here on GitHub. i.e. If you specify `ephemeralViolette` as the author then it must be submitted by @ephemeralViolette. Refer to the readme for detailed submission instructions.